app/components/sidebar/Menu.client.tsx

```diff
@@ -140,8 +140,7 @@ export const Menu = () => {
           >
             Problems
           </a>
-          <SaveProblem />
-          {isAdmin && <SaveReproductionModal />}
+          {isLoggedIn && <SaveProblem />}
```

Comment:
NOTE: Should we require admin access for saving a repro?

Reply:
Considering the SaveProblem now also includes SaveReproduction, yeah we should restrict this to admin users.